app/components/time_component.html.erb

```diff
@@ -1,3 +1,3 @@
 <%= time_tag time do %>
-  <%= time_ago_in_words time %> ago
```

Comment:
"ago" still needs to appear in the UI.

Reply:
I made it so ago appears, but it will be replaced by "auparavant" in French :)